README.md

```diff
@@ -60,7 +60,7 @@ By default, the `Xamarin.Forms.Controls` project does not have a configuration f
 
 You will have to obtain your own API keys for each of these services, inserted directly after the identifier (e.g. `UWPMapsAuthKey:abcdefghijklmnopqrstuvwxyz`). You can find out how to obtain each of these as follows:
 
-- `UWPMapsAuthKey` at https://www.microsoft.com/maps/create-a-bing-maps-key.aspx
+- `UWPMapsAuthKey` at https://microsoft.com/maps/create-a-bing-maps-key
```

Comment:
this updated URL doesn't take me to a page where I can create a key. The existing URL does. What am I missing here?